model_compression_toolkit/core/pytorch/reader/reader.py

```diff
@@ -139,12 +151,15 @@ def model_reader(model: torch.nn.Module,
         representative_data_gen (Callable): Dataset used for calibration.
         to_numpy: Function to convert framework's tensor to a Numpy array.
         to_tensor: Function to convert a Numpy array to a framework's tensor.
+        model_leaf_layers (list): List of the module's custom layers, these layers shouldn't be divided into
```

Comment:
List of the module's custom layers. These layers shouldn't be divided into their submodules. Please note that their quantization will not be optimized.